lib/geomancer.ex

```diff
@@ -1,21 +1,19 @@
 defmodule Geomancer do
-  @moduledoc """
-  A library to convert between various geospatial file formats
-  """
+  @moduledoc false
```

Comment:
Why remove the moduledoc?

Reply:
I'm going to do another pass at a later time to write moduledocs for everything. I didn't want to overlook this one that already has one, so I removed it. Kind of TODO'ing it, I guess.